traffic_portal/test/integration/PageObjects/UsersPage.po.ts

```diff
@@ -46,13 +68,39 @@ export class UsersPage extends BasePage {
     private txtPassword = element(by.name('uPass'));
     private txtConfirmPassword = element(by.name('confirmPassword'));
     private txtPublicSSHKey = element(by.name('publicSshKey'));
+    private txtSearch = element(by.id('usersTable_filter')).element(by.css('label input'));
+    private btnTableColumn = element(by.css('[title="Select Table Columns"]'));
     private randomize = randomize;
 
-    async OpenUserPage(){
-      let snp = new SideNavigationPage();
-      await snp.ClickUserAdminMenu();
-      await snp.NavigateToUsersPage();
-     }
+    async OpenUserPage() {
+        let snp = new SideNavigationPage();
+        await snp.NavigateToUsersPage();
```

Comment:
this works fine as-is, but if it's more intuitive for you, you should knowyou can just directly return this Promise to run asynchronously and resolve the Promise returned by this method when it resolves. It'll work the same way, essentially, since this method is `async`, but might make more sense since I asked for a return type for the function to actually `return` something explicitly. (To be clear, it is returning something implicitly already).